Cargo.toml

```diff
@@ -11,3 +11,8 @@ license = "MIT OR Apache-2.0"
 bevy-earcutr = "0.9"
 bevy = { version = "0.10", default-features = false,  features = ["bevy_render"] }
 geo = "0.25"
+geo-types = "0.7"
+
+[patch.crates-io]
+# TEMP: https://github.com/georust/geo/pull/1020 is yet to included in a release.
```

Comment:
https://github.com/georust/geo/pull/1022

Reply:
Actually https://github.com/georust/geo/pull/1023 is more relevant here

Reply:
Nice :) Ok if I just wait for #1023 to get merged and then remove the patch block all together?

Reply:
https://github.com/georust/geo/pull/1023 has merged and I just published a new geo-types version. So if you specify `geo-types = "0.7.10"` it should pick up this change

Reply:
Done!